service/src/main/java/bio/terra/workspace/app/controller/shared/PropertiesUtils.java

```diff
@@ -28,4 +34,16 @@ public static ApiProperties convertMapToApiProperties(Map<String, String> proper
     properties.forEach((key, value) -> apiProperties.add(new ApiProperty().key(key).value(value)));
     return apiProperties;
   }
+
+  /**
+   * Clear certain properties in the hashmap before making a clone.
```

Comment:
before cloning a resource

(clarifying this is for resource clone, not workspace clone)